language_service/src/completion/tests.rs

```diff
@@ -174,84 +174,6 @@ fn ignore_unstable_callable() {
     );
 }
 
-#[test]
-fn ignore_internal_callable() {
```

Comment:
Do we no longer ignore internal callables? That seems like an intended feature, no?

Reply:
Ah I see this is explained in your description, my bad. Is there a similar test somewhere that ensures that internal items in _other packages_ are not discoverable during completions?

Reply:
Yeah, that's verified via HIR tests [here](https://github.com/microsoft/qsharp/blob/7ae7a6f1b15f33aaf1c1fe6e81beaeb2fd55372c/compiler/qsc_frontend/src/compile/tests/multiple_packages.rs#L184). We don't have a test for that specifically in completions, though.